second_level_cache.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency "activesupport", ["> 4.0.0", "< 5.0"]
 
   gem.add_runtime_dependency "activerecord", ["> 4.0.0", "< 5.0"]
-  gem.add_development_dependency "sqlite3"
+  gem.add_development_dependency "sqlite3", "~> 1.3.6"
```

Comment:
When exec `bundle exec rake` in local, occur error:

```
$ bundle exec rake
/path/to/gems/activerecord-4.2.11.3/lib/active_record/connection_adapters/connection_specification.rb:177:in `rescue in spec': Specified 'sqlite3' for database adapter, but the gem is not loaded. Add `gem 'sqlite3'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord). (Gem::LoadError)
	from /path/to/gems/activerecord-4.2.11.3/lib/active_record/connection_adapters/connection_specification.rb:174:in `spec'
	from /path/to/gems/activerecord-4.2.11.3/lib/active_record/connection_handling.rb:50:in `establish_connection'
	from /path/to/second_level_cache/test/test_helper.rb:14:in `<top (required)>'
	from /path/to/second_level_cache/test/base_test.rb:2:in `require'
	from /path/to/second_level_cache/test/base_test.rb:2:in `<top (required)>'
	from /path/to/gems/rake-13.0.1/lib/rake/rake_test_loader.rb:17:in `require'
	from /path/to/gems/rake-13.0.1/lib/rake/rake_test_loader.rb:17:in `block in <main>'
	from /path/to/gems/rake-13.0.1/lib/rake/rake_test_loader.rb:5:in `select'
	from /path/to/gems/rake-13.0.1/lib/rake/rake_test_loader.rb:5:in `<main>'
rake aborted!
Command failed with status (1)
```

activerecord 4.x is depend on `sqlite3` gem is version 1.3.6
https://github.com/rails/rails/blob/4-2-stable/Gemfile#L104